URLTester/Program.cs

```diff
@@ -43,7 +43,7 @@ static void Main(string[] args)
             OutputProcessMessage(Messages.Running);
 
 
-            if (!testManager.TestLinks())
+            if (!testManager.TestLinks(OutputManager.WriteProgressToConsole))
```

Comment:
This is wrong.  You took the idea I had with using delegates to handle the output in other locations and jacked it up.  You created a whole new delegate.  The idea behind the Output manager was to be able to hot swap the output source t o anything like Database, Console, File, with minimal effort. By creating a new delegate you introduced confusing and complexity.